projects/addon-doc/src/components/example/example.template.html

```diff
@@ -42,27 +42,15 @@ <h3 *ngIf="heading" class="t-title">
             </button>
         </div>
 
-        <ng-container
-            *ngIf="{
-                activeTab: tabs[activeItemIndex],
-                code: content[tabs[activeItemIndex]] || ''
-            } as state"
-        >
-            <div
-                *ngIf="state.activeTab == defaultTab; else codeSection"
-                automation-id="tui-doc-example"
-                class="t-demo"
+        <!-- dynamic re-render tab content in runtime for trigger css animation -->
+        <ng-container *ngFor="let tab of tabs; let index = index">
```

Comment:
I believe this works only because the preview tab that shows `ng-content` is the first one in the list of `tabs`. Kind of a shaky solution just to retrigger an animation. We can probably just bind something like active index to an attribute and switch CSS animations based on that.

Reply:
fixed